tests/RunTests.cmd

```diff
@@ -178,6 +182,22 @@ echo nunit-console.exe /nologo /result=%XMLFILE% /output=%OUTPUTFILE% /err=%ERRO
 
 goto :EOF
 
+:COMPILERUNIT
```

Comment:
This unit test should be automatically run by CI but I am not sure where this is configured. The test suite is rather small (for now) it's quick.